app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -66,12 +68,25 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
       .catch(miqService.handleFailure);
   };
 
+  $scope.filterCloudNetworkChanged = function(id) {
+    miqService.sparkleOn();
+    $http.get('/network_router/network_router_subnets_by_network/' + id)
```

Comment:
I missed the fact that this is an RFE BZ for 5.7... so maybe the REST API implementation can wait for now.
Can you create a follow-up PR later to call the REST API?